opennms-dao/src/main/java/org/opennms/netmgt/dao/support/RrdResourceAttributeUtils.java

```diff
@@ -255,4 +256,79 @@ protected static String getStringProperty(File directory, String key) {
             throw new DataAccessResourceFailureException(message, e);
         }
     }
+
+    /**
+     * Alphanumeric sort that handles substrings with numeric components
+     */
+    public static class AlphaNumericOnmsAttributeComparator implements Serializable, Comparator<OnmsAttribute> {
```

Comment:
This is screaming at me for unit test coverage.

Reply:
> What's `BS`? :)

Fiiiine, it didn't seem worth making a whole other issue for a pretty simple change, but for you, I'll do it.  ;)

Yeah, I can add a couple unit tests.

Reply:
Didn't bother changing the branch name, but changed the commits.